### PR TITLE
[ADD] l10n_cl_currency_rate_sbif

### PR DESCRIPTION
**Pull request by max3903**
Update the exchanges rates of the currencies used in Chile using the SBIF web services.

### CODE REVIEW

**Comment by max3903** [comment]
Ping @nelsonramirezs

File: l10n_cl_currency_rate_sbif/views/update_button.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <field name="arch" type="xml">
+            <xpath expr="/form/sheet/div[@name='button_box']"
+                   position="inside">
+                <button string="Update Currency"
+                        type="object"
+                        name="action_update_currency"
+                        class="oe_stat_button"
+                        icon="fa-magnet"
+                        attrs="{'invisible':[('name', 'not in',
+                        ['SBIFUSD', 'SBIFUF', 'SBIFUTM', 'SBIFEUR'])]}"/>
```
Comment (bodedra) [inline l10n_cl_currency_rate_sbif/views/update_button.xml]: @max3903 `name` field takes 3 char only. It seems `Update Currency` button never visible. :thinking:

**Review by bodedra — APPROVED** [review]
Code review. LGTM :+1:

**Review by nelsonramirezs — COMMENTED** [review]
Tested. 
I think its missing the record

    <record model='ir.config_parameter' id='sbif_fin_apikey'>
        <field name='key'>sbif.financial.indicators.apikey</field>
        <field name='value'></field>
    </record>

Then Ok!

**Comment by max3903** [comment]
> Tested.
> I think its missing the record
> 
> ```
> <record model='ir.config_parameter' id='sbif_fin_apikey'>
>     <field name='key'>sbif.financial.indicators.apikey</field>
>     <field name='value'></field>
> </record>
> ```
> 
> Then Ok!

https://github.com/OCA/l10n-chile/pull/12/files#diff-cb45779cbbba6afb06bc16144efe4d44R5

File: l10n_cl_currency_rate_sbif/data/ir_config_parameter.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<odoo noupdate="1">
+
```
Comment (nelsonramirezs) [inline l10n_cl_currency_rate_sbif/data/ir_config_parameter.xml:2]:
```suggestion
    <record model='ir.config_parameter' id='sbif_fin_apikey'>
        <field name='key'>sbif.financial.indicators.apikey</field>
        <field name='value'></field>
    </record>
```

**Review by nelsonramirezs — COMMENTED** [review]
It works good, but I had to put the key "sbif.financial.indicators.apikey" manually. 
The value is the apikey.

**Review by nelsonramirezs — APPROVED** [review]

**Comment by max3903** [comment]
@nelsonramirezs Fixed and documentation updated.

**Review by nelsonramirezs — APPROVED** [review]

**Review by nelsonramirezs — APPROVED** [review]

**Pull request merged**